M2/Macaulay2/packages/FastLinAlg.m2

```diff
@@ -1381,8 +1462,7 @@ doc ///
         n1: ZZ
         M1: Matrix
     Outputs
-        : Boolean
-            true if it has that rank, otherwise false
+        : List
```

Comment:
The description doesn't say this function always returns a list.  Or ever returns a list.

Reply:
I think I fixed this.